webpack.common.js

```diff
@@ -16,10 +14,6 @@ if (fs.existsSync('config.json')) {
   config = require('./tests/drone/config.json')
 }
 
-if (fs.existsSync(`./themes/${config.theme}/favicon.jpg`)) {
-  favicon = `./themes/${config.theme}/favicon.jpg`
```

Comment:
It is for sure not the perfect solution to have favicon handling in webpack - but we need theme based favicons - please keep!

Reply:
Also, in the wake of supporting IE11, we may need to support multiple files here:
https://www.emergeinteractive.com/insights/detail/the-essentials-of-favicons/